<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <italic>true</italic>
    <bold>false</bold>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="Options">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>540</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="Start">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Stop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Continue">
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="End">
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Population">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>660</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Population</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="L_grass">
       <property name="text">
        <string>Grass</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="L_tiger">
       <property name="text">
        <string>Tiger</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="Grass_num">
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>3000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="Tiger_num">
       <property name="maximum">
        <number>301</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="L_sheep">
       <property name="text">
        <string>Sheep</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="L_cow">
       <property name="text">
        <string>Cow</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="Cow_num">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="Sheep_num">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>310</y>
     <width>151</width>
     <height>171</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use the dial to adjust the movement speed of creatures&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="singleStep">
    <number>0</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="Move_Speed">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>220</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>timeinterval  per frame</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="format">
    <string> %v ms / f</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Notation</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="notation_sheep">
      <property name="text">
       <string>1,2,3 represent the kid, adult, old states of sheep</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="notation_cow">
      <property name="text">
       <string>4,5 represent the kid, adult, old states of cow </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="notation_tiger">
      <property name="text">
       <string>7,8 represent the male and female states of tiger</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
